buildspecs/core.feature

```diff
@@ -115,6 +115,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="opt_newObjectHash" value="true"/>
 		<flag id="opt_newRomClassBuilder" value="true"/>
 		<flag id="opt_openjdkMethodhandle" value="false"/>
+		<flag id="opt_openjdkFfi" value="true"/>
```

Comment:
Why is this a flag? The code is guarded by a JCL level check.

Reply:
It was previously used for `ProgrammableInvoker` and its fields set in the vmconstantpool.xml to skip the compilation in OpenJ9 JCL code (`java.base`) but `ProgrammableInvoker` was moved to different module (not `java.base`) in which case the flag might be outdated. I need to compile without the flag to see whether it is still required or not.

Reply:
Removed the flag as it is not needed any more.

Reply:
@gacholio,  I reviewed previous discussion with Pete offline on Slack.    So the reason for the flag being added in the constant pool is to indicate that all entries with `ProgrammableInvoker`  in the `vmconstantpool.xml` are not part of `java.base` and should be skipped in `initializeKnownClasses`; otherwise, it goes wrong in multiple runs with the shared class enabled because its module is incorrectly set to `java.base` in `initializeKnownClasses` at startup after the flag was removed (tried another time and it ended with the exception as follows):
```
Exception in thread "main" java.lang.IllegalAccessError: Class jdk/internal/foreign/abi/x64/sysv/CallArranger(jdk.incubator.foreign) can not access class jdk/internal/foreign/abi/ProgrammableInvoker(java.base) because module java.base does not export package jdk/internal/foreign/abi to module jdk.incubator.foreign
	at jdk.incubator.foreign/jdk.internal.foreign.abi.x64.sysv.CallArranger.arrangeDowncall(CallArranger.java:135)
	at jdk.incubator.foreign/jdk.internal.foreign.abi.x64.sysv.SysVx64Linker.downcallHandle(SysVx64Linker.java:93)
```
In this case, the flag (for both UMA & Cmake) must be added back to ensure it works good as expected.

Reply:
Confirmed that the exception was gone with the flag enabled there after verifying the test cases many times.